documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -179,43 +282,50 @@ the group name. This will look like that in your connector JSON:
-Note that `replication.factor` and `partitions` properties are optional for custom groups. Custom
-groups will fallback to the `default` group's value if not set. Use `-1` to use the Kafka broker's
-default value.
+|Defines the configuration for the `applicationlogs` group.{empty} +
+The `replication.factor` and `partitions` properties are optional for custom groups. If no value is set,
+custom groups fall back to the value set for the `default` group. Set the value to `-1` to use the value that is set for the Kafka broker.
 
 |4
 |`topic.creation.applicationlogs.include` defines a regular expression to match all topics that start
-with `dbserver1.logs.applog-`. The config defined for the `applicationlogs` group will only be
-applied when the topic name matches the given regular expression. As there's also the `exclude`
-property defined at position *<5>* all topics matching this `include` regular expression might be
-further restricted by the that `exlude` property.
+with `dbserver1.logs.applog-`. The configuration that is defined for the `applicationlogs` group is
+applied only to topics with names that match the specified regular expression. Because an `exclude`
+property is also defined for this group, the topics that match the `include` regular expression might be
+further restricted by the that `exclude` property.
 
 |5
 |`topic.creation.applicationlogs.exclude` defines a regular expression to match all topics that start
-with `dbserver1.logs.applog-old-`. The config defined for the `applicationlogs` group will only be
-applied when the topic name does *not* match the given regular expression. As there's also the
-`include` property set for this group the `applicationlogs` group will only be applied to topics
-which name matches the `include` regular expression/s *and* _not_ match the `exclude` regular
-expression/s.
+with `dbserver1.logs.applog-old-`. The configuration that is defined for the `applicationlogs` group is
+applied only to topics with name that do *not* match the given regular expression. Because an
+`include` property is also definedfor this group, the configuration of the `applicationlogs` group is applied only to topics
+with names that match the specified `include` regular expressions *and* that do _not_ match the specified `exclude` regular
+expressions.
 |===
 
+// Type: procedure
+// ModuleID: registering-debezium-custom-topic-creation-groups
+// Title: Registering {prodname} custom topic creation groups
+[id="registering-custom-topic-creation-groups"]
 === Registering custom groups
 
-Finally, we need to register the two defined custom groups `inventory` and `applicationlogs` with
-the `topic.creation.groups` property:
+After you specify the configuration for any custom topic creation groups, register the groups.
+
+.Procedure
 
+* Register custom groups by adding the `topic.creation.groups` property to the connector JSON, and specifying a comma-separated list of groups.
```

Comment:
JSON should only be used upstream. Same comment below and potentially elsewhere in this doc.